tools/Environments/DevHome.Environments/CustomControls/CardFlyout.xaml.cs

```diff
@@ -27,25 +28,70 @@ public ObservableCollection<OperationsViewModel> ItemsViewModels
     private static void ItemsLoaded(DependencyObject dependencyObj, DependencyPropertyChangedEventArgs args)
     {
         var flyout = (CardFlyout)dependencyObj;
+
+        if (args.OldValue != null)
+        {
+            var oldOperationsViewModel = (INotifyCollectionChanged)args.OldValue;

```

Comment:
This'll crash if `args.OldValue` isn't an `INotifyCollectionChanged` is that okay in this context?